control-plane/subcommand/common/common.go

```diff
@@ -80,9 +82,9 @@ func ValidateUnprivilegedPort(flagName, flagValue string) error {
 // ConsulLogin issues an ACL().Login to Consul and writes out the token to tokenSinkFile.
 // The logic of this is taken from the `consul login` command.
 func ConsulLogin(client *api.Client, bearerTokenFile, authMethodName, tokenSinkFile, namespace string, meta map[string]string) error {
-	if meta == nil {
-		return fmt.Errorf("invalid meta")
-	}
+	//if meta == nil {
```

Comment:
If we decide to build a controller to cleanup this is where we'd input the requisite metadata to make searching for it fast.